docs/network-flow-visibility.md

```diff
@@ -40,6 +40,18 @@
       - [Flow Records](#flow-records)
       - [Node Throughput](#node-throughput)
       - [Network Policy](#network-policy)
+  - [Grafana Flow Collector](#grafana-flow-collector)
```

Comment:
Can we change the title of the `ELK Flow Collector` section to `ELK Flow Collector (deprecated)` as my understanding is that all the development effort moving forward will be for the Grafana Flow Collector? If this is accurate, I would also recommend switching the order of the sections so that `Grafana Flow Collector` comes before `ELK Flow Collector`.